### PR TITLE
Native tailwindcss purgecss integration

### PR DESCRIPTION
**Pull request by Jfelix61**
### Type: enhancement

Closes: #348 

### What are the changes and their implications? :gear:

Integrate tailwind's default purgecss instead of having it as a separate package.

### CODE REVIEW

File: examples/tailwind/package.json [hunk 1]
```diff
@@ -50,11 +49,10 @@
     "eslint-plugin-react-hooks": "3.0.0",
     "husky": "4.2.5",
     "lint-staged": "10.1.7",
-    "next": "9.3.5",
```
Comment (merelinguist) [inline examples/tailwind/package.json:53]: I believe we still need next installed as per the note above

**Comment by merelinguist** [comment]
Thank you for the great work here! 🥳

**Comment by flybayer** [comment]
🔥

**Comment by osirvent** [comment]
Since Nextjs is in `./app`, `tailwind.config.js` shouldn't reflect that?

```
module.exports = {
  purge: [
    "./app/pages/**/*.{js,jsx,ts,tsx}",
    "./app/components/**/*.{js,jsx,ts,tsx}"
  ],
  theme: {
```

**Comment by flybayer** [comment]
@osirvent yes, you're exactly right — looks like the example is probably broken right now. Want to open a PR or issue for this?

**Comment by osirvent** [comment]
Perfect, I'll do.

**Comment by osirvent** [comment]
#381

**Pull request merged**